lib/manager/bundler/readme.md

```diff
@@ -46,78 +36,86 @@ No
 
 #### Can package files have "local" links to each other that need to be resolved?
 
+No
+
 #### Is there reason why package files need to be parsed together (in serial) instead of independently?
 
----
+No
 
 #### What format/syntax is the package file in? e.g. JSON, TOML, custom?
 
----
+Ruby syntax
 
 #### How do you suggest parsing the file? Using an off-the-shelf parser, using regex, or can it be custom-parsed line by line?
 
----
+I'm thinking about to use something that can transform ruby code to abstract syntax tree
```

Comment:
I agree we should be properly parsing the Ruby code using Ruby. Let's see how using Bundler's API goes first, i.e. time/cpu/latency. If it's fast then we can just use it and return a JSON object to stdout for parsing from child_process in Node.js.

Reply:
👍